### PR TITLE
Fix Completion issues for current module suggestions and lang lib suggestions

### LINKED ISSUE #28155: Module prefixes for predeclared imports don't have to be quoted identifiers now
**Description:**
$title.

The current list of predeclared imports is https://github.com/ballerina-platform/ballerina-lang/blob/v2.0.0-alpha/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java#L373

**Steps to reproduce:**
Accepting 

langlib_import

adds

```ballerina
public function main() {
    'int:
}
```

But this doesn't have to be a quoted identifier now.

```ballerina
public function main() {
    int:
}
```

**Affected Versions:**
sl-alpha

### LINKED ISSUE #28191: When using modules inside same package, suggestions are not provided
**Description:**
If I am using some stdlib, plugin shows suggestions and auto import.
image

But If i am using module inside my project, module suggestions are not provided. ( Note: I am having a module called storage and try to access it from default module main.bal)

image

I has to type import winery/storage to fix.

image

### LINKED ISSUE #28411: Remote methods are sugggested when using field-access methods
**Description:**
Remote methods are suggested when using field-access methods. Also note the description should be corrected as `Client->delete()`.

Screenshot 2021-02-03 at 11 15 06 AM
 
**Steps to reproduce:**

**Affected Versions:**
2.0.0-alpha1

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by nadeeshaan**
## Purpose
Fixes #28191
Fixes #28155
Fixes #28411 

1. Add the completion support for the modules in the same project
2. Remove langlib suggestions since the predeclaration of the lang libs have been added.
3. Remove error keyword suggestion duplication.
4. Avoid suggesting remote methods on method access

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java [hunk 1]
```diff
@@ -215,6 +220,7 @@ public void sort(BallerinaCompletionContext context, T node, List<LSCompletionIt
                     String pkgName = importNode.moduleName().stream()
                             .map(Token::text)
                             .collect(Collectors.joining("."));
+                    String processedModuleHash = orgName.isEmpty() ? pkgName : orgName + "/" + pkgName;
```
Comment (IMS94) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java:223]: We can use `CommonKeys#SLASH_KEYWORD_KEY` for the `"/"` right?

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java [hunk 2]
```diff
@@ -223,56 +229,40 @@ public void sort(BallerinaCompletionContext context, T node, List<LSCompletionIt
...
-        // TODO: Refactor to match the latest project structure
-//        packages.addAll(LSPackageLoader.getCurrentProjectModules(currentPkg, ctx));
         packages.forEach(pkg -> {
             String name = pkg.packageName().value();
             String orgName = pkg.packageOrg().value();
-            if (!CommonUtil.matchingImportedModule(ctx, pkg) && !populatedList.contains(orgName + "/" + name)
-                    && !this.isPreDeclaredLangLib(pkg)) {
-                CompletionItem item = new CompletionItem();
-                item.setLabel(CommonUtil.getPackageLabel(pkg));
+            if (!CommonUtil.matchingImportedModule(ctx, pkg) && !processedList.contains(orgName + "/" + name)) {
```
Comment (IMS94) [inline language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java:241]: Same as above. We can use CommonKeys#SLASH_KEYWORD_KEY for the "/" right?

**Review by IMS94 — APPROVED** [review]

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java (hunk 2 above)
Comment (nadeeshaan) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java]: Fixed

**Review by rasika — APPROVED** [review]
LGTM

**Pull request merged**